pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,30 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	// +optional
+	Config *PrometheusConfiguration `json:"config,omitempty"`
+}
+
+// +k8s:openapi-gen=true
+// Config  is the the desired state of Prometheus Configuration
+type PrometheusConfiguration struct {
+
+	// user can mount prometheus rule config with external configMap.If use this feature, the external configMap must contain `prometheus-config` key in data.
+	ConfigMapRef *ConfigMapRef `json:"configMapRef,omitempty"`
+
+	// user can  use it specify prometheus command options
+	CommandOptions []string `json:"command,omitempty"`
```

Comment:
This seems wrong. The json is `command` but in the previous example this field is `commandOptions`

Reply:
done